app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewState.kt

```diff
@@ -1,10 +1,36 @@
 package com.wire.android.ui.home.conversations
 
+import android.os.Parcelable
 import androidx.compose.ui.text.input.TextFieldValue
 import com.wire.android.ui.home.conversations.model.Message
+import kotlinx.parcelize.Parcelize
 
 data class ConversationViewState(
     val conversationName: String = "",
     val messages: List<Message> = emptyList(),
-    val messageText : TextFieldValue = TextFieldValue("")
+    val messageText: TextFieldValue = TextFieldValue("")
 )
+
+/**
+ * Represents an attachment part of a message to be sent
+ */
+@Parcelize
+data class AttachmentPart(val mimeType: String, val rawContent: ByteArray) : Parcelable {
```

Comment:
I am abit confused about the name of this class :P AttachmentPart makes me think it is a part of something but part of what :P maybe just Attachment would be less confusing ? Dunno just a personal opinion I guess ;p

Reply:
How about renaming it to `AttachmentParcel` or `AttachmentBundle` ?